tools/picard.wdl

```diff
@@ -134,7 +134,7 @@ task validate_bam {
             GREP_PATTERN="(ERROR|WARNING)"
         fi
 
-        if [ ! ~{succeed_on_errors} ] \
```

Comment:
Unrelated change to the WDL version upgrade. This expression was _always_ evaluating to False. Fixed here because I noticed ShellCheck warning us about this Bashism while looking through the `miniwdl check` logs.